src/app/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,13 @@
+<button mat-button class="new-category-button" (click)="addCategory('')">
+  <div>New Category</div>
+  <mat-icon>add</mat-icon>
+</button>
+<div class="categories-holder" *ngIf="categories">
+  <div class="category" *ngFor="let category of categories.slice().reverse()">
```

Comment:
It's ill advised to use function calls in structural directives and property binding.

Please read this article for more information:
https://javascript.plainenglish.io/angular-dont-call-a-function-inside-the-template-anymore-e74ebf499bb8

If I'm not mistaken, you simply want to reverse the list of categories? If so, you can accomplish this using CSS and flexbox. I recommend taking that approach.